sdk/eventhub/event-hubs/src/eventProcessor.ts

```diff
@@ -216,7 +223,20 @@ export class EventProcessor {
             partitionId: partitionId
           };
 
-          const checkpointManager = new CheckpointManager();
+          for (const ownership of partitionOwnership) {
```

Comment:
`partitionsToAdd` gives you a list of partitions we aren't currently processing.

I think to keep things simple until we add load-balancing, for now you could just create a `PartitionOwnership` for each partition in `partitionsToAdd`, and then claim it. Either claim them all at once, or claim them one at a time, but you shouldn't need to nest a 2nd for loop under the 1st one.

Reply:
Yeah.. I don't know why I added that loop.

Updated the code and now claiming ownership for all the partition in partitionsToAdd.